documentation/content/en/articles/committers-guide/_index.adoc

```diff
@@ -2019,6 +2019,91 @@ git log refs/freebsd/vendor/zlib/1.2.10
 ....
 
 would look at the log for the vendor branch for zlib starting at 1.2.10.
+=== Collaborating with Others
+
+One of the keys to good software development on a project as large as FreeBSD is the ability to collaborate with others before you push your changes to the tree.  The FreeBSD project's git repositories do not, yet, allow user created branches to be pushed to the repository, and therefore if you wish to share your changes with others you must use another mechanism, such as a hosted GitLab or GitHub, in order to share changes in a user generated branch.
+
+The following instructions show how to set up a user generated branch, based on the FreeBSD main branch, and push it to GitHub.
+
+Before you begin, make sure that your local git repo is up to date and has the correct origins set as shown above.
+
+[source,shell]
+````
+% git remote -v
+freebsd https://git.freebsd.org/src.git (fetch)
+freebsd ssh://git@gitrepo.freebsd.org/src.git (push)
+````
+
+The first step is to create a fork of https://github.com/freebsd/freebsd-src[FreeBSD] on GitHub.
+The destination of the fork should be your own, personal, GitHub account (gvnn3 in my case).
+
+Now add a remote that points to your fork:
```

Comment:
Do you need to have something more specific here "add a remote on your local machine that points to your GitHub fork"? Or is there enough context in the reader's mind here already that would make something more specific just more verbose?